v2/examples/cluster_aws.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"context"
+
+	"github.com/mongodb-forks/digest"
+	client "go.mongodb.org/atlas/mongodbatlas/v2"
+	apiv2latest "go.mongodb.org/atlas/mongodbatlas/v2/api"
+
+	utils "go.mongodb.org/atlas/mongodbatlas/v2/test/utils"
+)
+
+/*
+* MongoDB Atlas Go SDK Example
+*
+* Example performs following actions:
+* - Finds first project in the user organization
+* - Creates new MongoDB cluster backed by AWS provider in that project
+* - Creates database user for connection
+* - Creates IP access to enable connection
+ */
+func main() {
```

Comment:
Based on your description the goal of this is for testing, two options, make this e2e tests, see the CLI for example how it's organize https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/clusters_file_test.go or consider proper [go examples](https://go.dev/blog/examples)

Moving to proper test removes the complexity of the workspace as test will be flagged as such and not included in final builds

Reply:
The intention was always to have it as an end-user example. 
We will compile it. Users can copy and paste the code to their repository etc.
We have separate tests that we will likely now move to Atlas CLI.